### ISSUE TITLE
Primary bookmark button on Stop Controller is broken in VoiceOver mode

### ISSUE THREAD

**Issue by aaronbrethorst** [OneBusAway/onebusaway-iphone#1434]
What I have found in using VO mode is that the Add Bookmark and Filter buttons are apparently being interpreted as being a single item, and double-tapping on them does nothing. 

From a user:

> Using the voiceover screen reader on an iPhone, I cannot add a bus stop as a bookmark. The button does not activate when I double tap on it which is the standard gesture to activate a button.
